beacon-chain/slasher/chunks_test.go

```diff
@@ -239,6 +283,49 @@ func TestMaxSpanChunksSlice_CheckSlashable(t *testing.T) {
 	require.NotEqual(t, (*ethpb.AttesterSlashing)(nil), slashing)
```

Comment:
I think this assert is not valid any more:
If `slashing` is `nil` but with  `(*ethpb.AttesterSlashingElectra)` type, then this `require.NotEqual` will be successful.